concepts/05 UI Components/PivotGrid/00 Getting Started with PivotGrid/08 Configure Fields and Areas.md

```diff
@@ -0,0 +1,135 @@
+To display data in the PivotGrid, specify the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array. Each object in it configures a single pivot grid field. Specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.
+
+You can distribute fields between four different areas: row, column, filter, and data. Row and column fields contain values displayed in row and column headers. Filter fields has no visual representation, but they can be used to filter data. Data fields contain values for summary calculation. To specify the area, set the [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property. In the following code, we declare two row fields, two column fields, and two data fields:
```

Comment:
> Filter fields **has** no visual **representation**, but they can be used to filter data.

1. Not "has" but "have".
2. I've grown to fear the root "present" in our texts, moreover in the "represent" form.

 Can we rewrite this sentence as follows?
 
 > Filter fields are invisible but you can filter data against them.